framework/include/task_manager/task_manager.h

```diff
@@ -134,6 +134,11 @@ typedef void (*_tm_unicast_t)(tm_unicast_msg_t *);
  */
 typedef void (*_tm_broadcast_t)(void *);
 
+/**
+ * @brief Termination callback function type
+ */
+typedef void (*_tm_termination_t)(void *);
```

Comment:
how about merging typedef void (*_tm_broadcast_t)(void *) and typedef void (*_tm_termination_t)(void *)?